dataland-frontend/src/components/resources/landing/SampleSection.vue

```diff
@@ -5,10 +5,10 @@
         <div class="grid">
           <div class="col-10 text-left text-gray-100">
             <h2 class="mb-0">Explore Dataland</h2>
-            <h2 class="mt-0">No registration required.</h2>
+            <h2 class="mt-0">No preview account required.</h2>
           </div>
           <div class="col-10 text-left text-gray-100">
-            <p>Preview EU Taxonomy Data of <strong>BMW</strong>, before joining Dataland.</p>
+            <p>View the EU Taxonomy Data of <strong>BMW</strong>, before creating a preview account.</p>
```

Comment:
Should we change "account" to "preview account" in our email texts too?
Or is it ok if it says only "account" there?

Reply:
ok - changed account-> preview account in e-mail template